tests/Endpoints/IdesTest.php

```diff
@@ -32,7 +33,29 @@ public function testGetAllIdes(): void
         $this->assertNotEmpty($result);
 
         foreach ($result as $record) {
-            $this->assertInstanceOf('\AcquiaCloudApi\Response\IdeResponse', $record);
+            $this->assertInstanceOf(IdeResponse::class, $record);
+
+            foreach ($this->properties as $property) {
+                $this->assertObjectHasAttribute($property, $record);
+            }
+        }
+    }
+
+    public function testGetMyIdes(): void
+    {
+
+        $response = $this->getPsr7JsonResponseForFixture('Endpoints/Ides/getAllIdes.json');
```

Comment:
I copied the examples of the other tests but it seems like they don't really.. test anything? I mean they load a json file and then ensure that json output is returned by the client, but I don't know what that really proves. For instance you can pass any parameter to `$ide->get()` in the test above and it will still pass. 🤷